src/domain/app/i18n/sv.json

```diff
@@ -693,6 +695,10 @@
         "general": "Evenemangsbeskrivning {{langText}}",
         "volunteering": "Volontärarbetetsbeskrivning {{langText}}"
       },
+      "labelHasEnvironmentalCertificate": "Event has received an EcoCompass or similar environmental certification",
```

Comment:
Should this be in Swedish?

Reply:
Translations: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/185/commits/9d8ee2af93331ff695811a6a1d684ae18200167c